src/prx/test/test_helpers.py

```diff
@@ -144,6 +144,72 @@ def test_ecef_to_geodetic():
     assert np.abs(expected_geodetic[2] - computed_geodetic[2]) < tolerance_alt
 
 
+def test_geodetic_2_ecef():
+    tolerance_ecef = 1e-3
+
+    expected_ecef = [6378137.0, 0.0, 0.0]
```

Comment:
Would it make sense here to put 

```
 computed_ecef = helpers.geodetic_2_ecef(
        expected_geodetic[0], expected_geodetic[1], expected_geodetic[2]
    )
    assert (
        np.linalg.norm(np.array(expected_ecef) - np.array(computed_ecef))
        < tolerance_ecef
    )
```

into a loop or helper function?

Reply:
Done!